src/genie/libs/parser/iosxe/tests/ShowCtsRoleBasedPermissions/cli/equal/golden_output_output.txt

```diff
@@ -0,0 +1,26 @@
+IPv4 Role-based permissions default:
+        Permit IP-00
```

Comment:
do you know if any chance to have multiple like below? if yes, we might need to consider to update schema for it.
```
IPv4 Role-based permissions default:
        Permit IP-00
        Permit IP-01

Reply:
From all the examples I saw, it won't have multiple like the example you showed. Because of that, I left the schema the same.